larray_editor/arrayadapter.py

```diff
@@ -92,8 +101,9 @@ def set_data(self, data, bg_gradient=None, bg_value=None, current_filter=None):
         if current_filter is None:
             self.current_filter = {}
         self.la_data = la.aslarray(data)
+        self.bg_value = la.aslarray(bg_value) if bg_value is not None else None
+        self.bg_gradient = bg_gradient
```

Comment:
I do not think storing/handling the **bg_gradient** in the adapter adds any value.

Reply:
Well, the changes made in PR #64 works fine as long as we do play with user defined gradient. 
In case of user defined gradient, it is overwritten as soon as we call the method reset_minmax() from DataArrayModel. 
In the current commit, bg_gradient is first overwritten by reset_minmax() and then reset to the user defined gradient but the call to `self.data_model.set_background(self.bg_gradient, bg_value_2D)` at the end of `update_filtered_data` in the Adapter. This is bad, I know...

Reply:
Ok, that will do for now. I will fix that in the "gradient combobox" branch I just spoke about because I need to touch that code anyway.